content/hi/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -21,66 +21,66 @@ sitemap:
 
-**वर्चुअलाइज्ड डिप्लॉयमेंट युग:**  एक समाधान के रूप में, वर्चुअलाइजेशन पेश किया गया था। यह आपको एक भौतिक सर्वर के सीपीयू (CPU) पर कई वर्चुअल मशीन (वीएम) चलाने की अनुमति देता है। वर्चुअलाइजेशन एप्लिकेशनो को वीएम (VM) के बीच अलग-थलग करने की अनुमति देता है और सुरक्षा का एक स्तर प्रदान करता है क्योंकि एक एप्लिकेशन की जानकारी को दूसरे एप्लिकेशन द्वारा स्वतंत्र रूप से एक्सेस नहीं किया जा सकता है।
+**वर्चुअलाइज्ड डिप्लॉयमेंट युग:**  एक समाधान के रूप में, वर्चुअलाइजेशन पेश किया गया था। यह आपको एक भौतिक सर्वर के सीपीयू (CPU) पर कई वर्चुअल मशीन (वीएम) चलाने की अनुमति देता है। वर्चुअलाइजेशन एप्लिकेशनो को वीएम (VM) के बीच  अलगाव उपलब्ध करता है और सुरक्षा का एक स्तर प्राप्त करता है क्योंकि एक एप्लिकेशन की जानकारी को दूसरे एप्लिकेशन द्वारा स्वतंत्र रूप से प्राप्त नहीं किया जा सकता है।
 
-वर्चुअलाइजेशन एक भौतिक सर्वर में संसाधनों के बेहतर उपयोग की अनुमति देता है और बेहतर मापनीयता की अनुमति देता है क्योंकि एक एप्लिकेशन को आसानी से जोड़ा या अपडेट किया जा सकता है, हार्डवेयर लागत को कम करता है, और बहुत कुछ। वर्चुअलाइजेशन के साथ आप भौतिक संसाधनों का एक सेट डिस्पोजेबल वर्चुअल मशीनों के समूह के रूप में प्रस्तुत कर सकते हैं।
+वर्चुअलाइजेशन एक भौतिक सर्वर में संसाधनों के बेहतर उपयोजन की अनुमति देता है और बेहतर मापक्रमणीयता की अनुमति देता है क्योंकि एक एप्लिकेशन को आसानी से जोड़ा या नवीनीकरण किया जा सकता है, यंत्रसामग्री के लागत को कम करता है, और बहुत कुछ। वर्चुअलाइजेशन के साथ आप भौतिक संसाधनों का एक सेट प्रयोगशील वर्चुअल मशीनों के समूह के रूप में प्रस्तुत कर सकते हैं।
 
-प्रत्येक वीएम वर्चुअलाइज्ड हार्डवेयर के शीर्ष पर अपने स्वयं के ऑपरेटिंग सिस्टम सहित सभी घटकों को चलाने वाली एक पूर्ण मशीन है।
+प्रत्येक वर्चुअल मशीन एक पूरी मशीन की तरह होती है, जिसमें अपने ऑपरेटिंग सिस्टम के साथ सभी घटक भी होते हैं, और ये सब वर्चुअलाइज्ड हार्डवेयर पर चलते हैं।
 
-**कंटेनर डिप्लॉयमेंट युग:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनो के बीच ऑपरेटिंग सिस्टम (OS) को साझा करने के लिए अलगाव गुण होते हैं। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। चूंकि वे अंतर्निहित बुनियादी ढांचे से अलग हो गए हैं, वे बादलों और ओएस (OS) वितरण में पोर्टेबल हैं।
+**कंटेनर डिप्लॉयमेंट युग:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनो के बीच ऑपरेटिंग सिस्टम (OS) को साझा करने के लिए एकाकीपन की स्वतंत्रता होती है। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। चूंकि वे अंतर्निहित अवसंरचना से अलग हो गए हैं, वे बादलों और ओएस (OS) वितरण में सुवाह्य  हैं।
```

Comment:
```suggestion
**कंटेनर डिप्लॉयमेंट युग:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनो के बीच ऑपरेटिंग सिस्टम को साझा करने के लिए एकाकीपन की स्वतंत्रता होती है। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। चूंकि वे अंतर्निहित अवसंरचना से अलग हो गए हैं, वे क्लाउड और ऑपरेटिंग सिस्टम वितरण में पोर्टेबल हैं।
```

Reply:
portable too is  english when सुवाह्य is quite used word in hindi

Reply:
Same answer as in above comment, and "सुवाह्य" is not a commonly used word.